pkg/monitor/api.go

```diff
@@ -128,6 +132,44 @@ func StartOpenShiftAPIMonitoring(ctx context.Context, m *Monitor, clusterConfig
 	return nil
 }
 
+func StartOAuthAPIMonitoring(ctx context.Context, m *Monitor, clusterConfig *rest.Config, timeout time.Duration) error {
```

Comment:
1:1 copy of the openshift-apiserver one, just with another resource?

Reply:
yes :)